packages/wrangler/src/git-client.ts

```diff
@@ -125,8 +127,24 @@ export async function cloneIntoDirectory(
 	// cleanup: move the template to the target directory and delete `.git`
 	try {
 		fs.renameSync(templatePath, targetDirectory);
-	} catch {
-		throw new Error(`Failed to find "${subdirectory}" in ${remote}`);
+	} catch (err) {
+		// @ts-expect-error non standard property on Error
+		if (err.code !== "EXDEV") {
+			logger.debug(err);
+			throw new Error(`Failed to find "${subdirectory}" in ${remote}`);
+		}
+		// likely on a different filesystem, so we need to copy instead of rename
+		// and then remove the original directory
+		try {
+			fs.cpSync(templatePath, targetDirectory);
```

Comment:
We actually need a `recursive` option on this command too, which is why the tests were failing.
I have fixed this up and rebased.